Sources/AccountSDKIOSWeb/Lib/Storage/Keychain/KeychainSessionStorage.swift

```diff
@@ -1,9 +1,11 @@
 import Foundation
 
 internal class KeychainSessionStorage: SessionStorage {
+    private(set) var accessGroup: String?
```

Comment:
Didnt we update this to a computed property. In the last pr?

Reply:
Rebased :)